src/BloomBrowserUI/bookEdit/js/bloomEditing.js

```diff
@@ -440,22 +440,23 @@ function MakeSourceTextDivForGroup(group) {
   else {
     $(divForBubble).remove();//no tabs, so hide the bubble
     return;
-  }
+    }
+
+    var showEvents = false;
+    var hideEvents = false;
+    var shouldShowAlways = true;
+
+  
+    if(checkMightCauseHorizontallyOverlappingBubbles(group)) {
```

Comment:
The section around 935 calls MakeHelpBubble(), which in turn also calls checkMightCauseHorizontallyOverlappingBubbles(), so it's all one code path.